docs/src/components/twitch.stories.mdx

```diff
@@ -13,11 +13,21 @@ Display a Twitch video by including the component in your `.mdx`. The `twitchId`
 ## Basic usage
 
 <Preview>
-  <Twitch twitchId="612049948" />
+  <Twitch twitchId="679474192" parent="gatsby-mdx-embed.netlify.app" />
```

Comment:
@zslabs Thanks for this! Is this why in the [docs](https://gatsby-mdx-embed.netlify.app/?path=/docs/components-twitch--page) the Twitch videos stopped working? is it a new Twitch thing?

Reply:
Yep that's why! It looks like it was changed in Feb of this year https://discuss.dev.twitch.tv/t/twitch-embedded-player-updates-in-2020/23956

Reply:
Ah ha! Brilliant thanks for sorting it. I'll get it merged and do a release in the morning. Great stuff. Thanks again!